json.wake

```diff
@@ -0,0 +1,1305 @@
+package json
+
+from wake import _
+from utils import prefixError
+from query import _
+
+# Anyone who depends on json will also
+# need query functions.
+from query export def single query queryExists queryOne queryOptional queryLast queryLastOptional
+from query export def qPass qExtern qExternPass qSingle qSinglePass qIf qEditIf qFlatten qEditFlatten qAt qEditAt qDefault
+from query export type QueryError
+from query export def QueryError makeQueryError
+
+## Helpers that will soon be in the standard library. This will not be exported here.
```

Comment:
1) Did we ever add this to wake core?
2) Should we make a list.wake in wake-extras?

Reply:
1. I don't think so
2. single could arguably just be in the stdlib
3. bonus: so should `identity`

Reply:
That can be a follow up

Reply:
oh this is a different function. Yeah lookup could be in wake-extras/list.wake